src/components/PDFView/index.native.js

```diff
@@ -33,34 +35,42 @@ class PDFView extends Component {
             shouldAttemptPdfLoad: true,
             shouldShowLoadingIndicator: true,
             isPasswordInvalid: false,
+            failedToLoadPDF: false,
             password: '',
         };
         this.initiatePasswordChallenge = this.initiatePasswordChallenge.bind(this);
         this.attemptPdfLoadWithPassword = this.attemptPdfLoadWithPassword.bind(this);
         this.finishPdfLoad = this.finishPdfLoad.bind(this);
+        this.handleFailureToLoadPDF = this.handleFailureToLoadPDF.bind(this);
     }
 
     componentDidUpdate() {
         this.props.onToggleKeyboard(this.props.isShown);
     }
 
+    handleFailureToLoadPDF(error) {
+        if (error.message.match(/password/i)) {
+            this.initiatePasswordChallenge();
+            return;
+        }
+
+        this.setState({
+            failedToLoadPDF: true,
+            shouldAttemptPdfLoad: false,
```

Comment:
Should we go with PDF or Pdf? I'm inclined for PDF

Reply:
> we should add this in our style guide

Good call, this made me check our current style guide and the Airbnb style which we use as a base, and the latest has an entry on acronyms: https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms

![image](https://user-images.githubusercontent.com/87341702/206558554-39bd9aa0-ea9b-4a5d-97cb-290a55d7c8b2.png)

I guess we are back at all uppercase because it is in the style guideline we follow